src/amo/components/CollectionManager/index.js

```diff
@@ -118,25 +143,23 @@ export class CollectionManagerBase extends React.Component<Props, State> {
   }
 
   render() {
-    const { collection, errorHandler, i18n, siteLang } = this.props;
-
-    let collectionUrlPrefix = '';
-    if (collection && siteLang) {
```

Comment:
We have an `if` statement above that will prevent the code from getting this far if there is no `collection` (and we're editing) and if there is no `siteLang`, so this statement seemed superfluous.